src/Language/PureScript/Interactive/Types.hs

```diff
@@ -57,27 +62,35 @@ data PSCiState = PSCiState
   [ImportedModule]
   [P.Declaration]
   [(P.Module, P.ExternsFile)]
+  (P.ModuleName, P.Ident)
   P.Imports
   P.Exports
   deriving Show
 
 psciImportedModules :: PSCiState -> [ImportedModule]
-psciImportedModules (PSCiState x _ _ _ _) = x
+psciImportedModules (PSCiState x _ _ _ _ _) = x
 
 psciLetBindings :: PSCiState -> [P.Declaration]
-psciLetBindings (PSCiState _ x _ _ _) = x
+psciLetBindings (PSCiState _ x _ _ _ _) = x
 
 psciLoadedExterns :: PSCiState -> [(P.Module, P.ExternsFile)]
-psciLoadedExterns (PSCiState _ _ x _ _) = x
+psciLoadedExterns (PSCiState _ _ x _ _ _) = x
+
+psciInteractivePrint :: PSCiState -> (P.ModuleName, P.Ident)
+psciInteractivePrint (PSCiState _ _ _ x _ _) = x
 
 psciImports :: PSCiState -> P.Imports
-psciImports (PSCiState _ _ _ x _) = x
+psciImports (PSCiState _ _ _ _ x _) = x
 
 psciExports :: PSCiState -> P.Exports
-psciExports (PSCiState _ _ _ _ x) = x
+psciExports (PSCiState _ _ _ _ _ x) = x
 
 initialPSCiState :: PSCiState
-initialPSCiState = PSCiState [] [] [] nullImports primExports
+initialPSCiState = PSCiState [] [] [] initialInteractivePrint nullImports primExports
+
+-- | The default interactive print function.
+initialInteractivePrint :: (P.ModuleName, P.Ident)
+initialInteractivePrint = (P.moduleNameFromString "PSCI.Support", P.Ident "eval")
```

Comment:
Should this be `supportModuleName`? Otherwise I think that binding should be removed since I don't think we'd want to reference it.

Reply:
Oh actually, we do still use the `supportModuleName` binding, for the check during startup that `PSCI.Support` is defined. For now, I've left this check as-is, since I haven't implemented handling the `:print` directive in `.purs-repl` yet, which means that the printing function is always going to be `PSCI.Support.eval` to start with, and so it still makes sense to check that it is there (I think). I'll push a commit which removes the duplication.